Gruntfile.js

```diff
@@ -305,6 +305,26 @@ module.exports = function (grunt) {
         }())
       }
     },
+  
+    search: {
+      noAbsoluteUrlsInCss: {
+        files: {
+          src: ['assets/sass/**/*.scss']
+        },
+        options: {
+          searchString: /url\([\'\"]\//g,
```

Comment:
Is it ever the case that we do not have a ' or " before the url, i.e. url(/foo/bar) ? Just checking - I have a hunch we already have a lint rule for that.